src/changelogs/fetch_changelogs.py

```diff
@@ -0,0 +1,69 @@
+import os
+from os import listdir
+from os.path import isfile, join
+import feedparser
+from bs4 import BeautifulSoup
+from urllib import request
+
+
+class ChangelogFetcher:
+    def __init__(self):
+        self.changelogs_by_date = {}
+
+    def get_rss(self, rss_url, update_Existing = False):
+        self.RSS_URL = rss_url
+        self._fetch_forum_changelogs(update_Existing)
+        return self.changelogs_by_date
+
+    def get_txt(self, changelog_path, update_Existing = True):
```

Comment:
unused argument

Reply:
removed it, it can always be added and for now the files someone creates should always be loaded